pages/forms/TripForm/index.js

```diff
@@ -0,0 +1,77 @@
+import useSWR from "swr";
+import CreateButton from "@/components/CreateButton";
+import CancelButton from "@/components/CancelButton";
+
+export default function TripForm() {
+  const { mutate } = useSWR("/api/trips");
+
+  async function handleAddTrip(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const tripData = Object.fromEntries(formData);
+
+    const newTrip = {
+      title: tripData.title,
+      location: [{ country: tripData.country, city: tripData.city }],
+      timePeriod: [
+        { startDate: tripData.startDate, endDate: tripData.endDate },
+      ],
+    };
+
+    const response = await fetch("/api/trips", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(newTrip),
+    });
+
+    if (!response.ok) {
+      console.error(response.status);
+      return;
+    }
+
+    mutate();
+    event.target.reset();
```

Comment:
You do not need the `reset()` here, as you are redirecting the user to another page afterwards. If the user navigates back to the form, all fields will be cleared anyway. 🙂 
```suggestion
```